src/fh-family-experience-web/ViewModels/Extensions/FamilyHubViewModelExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FamilyHubs.ServiceDirectory.Shared.Models.Api.OpenReferralLocations;
+using FamilyHubs.ServiceDirectory.Shared.Models.Api.OpenReferralServices;
+using FamilyHubs.ServiceDirectory.Shared.Models;
+using System.Linq;
+
+namespace fh_family_experience_web.ViewModels.Extensions
+{
+    public static class FamilyHubViewModelExtensions
+    {
+        public static IList<FamilyHubViewModel> ToFamilyHubViewModels(this IList<Either<OpenReferralServiceDto, OpenReferralLocationDto, double>> hubsAndServices)
+        {
+            var results = new List<FamilyHubViewModel>();
+
+            if(hubsAndServices != null)
+            {
+                foreach(var location in hubsAndServices.Where(h => h.Second != null).OrderBy(l => l.Third))
```

Comment:
Hmm, I suspect `Second` _can_ be nullable, but it's advertised as non nullable.
It's being imported from the service directory shared nuget though.
Leave for now, and we can potentially look at the shared nuget later.